cylc/flow/task_job_mgr.py

```diff
@@ -1286,6 +1286,12 @@ def _prep_submit_task_job(
                     # bad_hosts:
                     self.bad_hosts -= exc.hosts_consumed
                     self._set_retry_timers(itask, rtconfig)
+                # Provide dummy platform otherwise it will show as localhost
+                # in the data store:
```

Comment:
Context easily lost, worth linking this in:

```suggestion
                # in the data store:
                # see https://github.com/cylc/cylc-flow/pull/6926
```